plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -141,23 +143,29 @@ private void setupDataBinding() {
   }
 
   private void setupAccountEmailDataBinding(DataBindingContext context) {
-    final IObservableValue accountEmailModel = PojoProperties.value("accountEmail").observe(model);
-    context.bindValue(new AccountSelectorObservableValue(accountSelector), accountEmailModel);
-
-    if (requireValues) {
-      context.addValidationStatusProvider(new FixedMultiValidator() {
-        @Override
-        protected IStatus validate() {
-          String email = (String) accountEmailModel.getValue();
-          // It's possible that no account is selected while a valid email has been saved in the
-          // model (if the corresponding account is signed out), so check the actual selection too.
-          if (email.isEmpty() || accountSelector.getSelectedEmail().isEmpty()) {
-            return ValidationStatus.error(Messages.getString("error.account.missing"));
-          }
-          return ValidationStatus.ok();
+    IValidator accountSelectedChecker = new IValidator() {
+      @Override
+      public IStatus validate(Object value /* email */) {
+        if (requireValues && Strings.isNullOrEmpty((String) value)) {
+          return ValidationStatus.error(Messages.getString("error.account.missing"));
         }
-      });
-    }
+        return ValidationStatus.ok();
+      }
+    };
+    UpdateValueStrategy targetToModel =
+        new UpdateValueStrategy().setBeforeSetValidator(accountSelectedChecker);
+    UpdateValueStrategy modelToTarget =
+        new UpdateValueStrategy().setBeforeSetValidator(accountSelectedChecker);
```

Comment:
You may want to change the formatting here to cascade the `.setConverter()` call to be consistent with the other calls.